<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQSettingsDialog</class>
 <widget class="QDialog" name="QQSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="globalPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>178</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Global</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelMaxHistorySize">
         <property name="text">
          <string>Max history size :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="maxHistLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelDefaultUA">
         <property name="text">
          <string>Default User-Agent :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="defaultUALlineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelDefaultLogin">
         <property name="text">
          <string>Default login :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="defaultLoginLineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="totozPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>178</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Totoz</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelSrvTotoz">
         <property name="text">
          <string>Totoz server :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="srvTotozLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelSrvAllowSearch">
         <property name="text">
          <string>Server allow search :</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelSrvQueryPattern">
         <property name="text">
          <string>Server query pattern :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="srvAllowSearch">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="srvTotozQueryPatternLineEdit"/>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="totozModeComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>En ligne</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>En popup</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>désactivé (en chauve)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelModeTotoz">
         <property name="text">
          <string>Totoz visual mode :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="srvTotozBaseImgLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Totoz base img URL:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="boardsPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>178</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Boards</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListView" name="bouchotListView"/>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="addBouchotButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../rc/tribunesDef.qrc">
              <normaloff>:/img/add-icon.png</normaloff>:/img/add-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteBouchotButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../rc/tribunesDef.qrc">
              <normaloff>:/img/delete-icon.png</normaloff>:/img/delete-icon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editBouchotButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../rc/tribunesDef.qrc">
              <normaloff>:/img/menu-edit-icon.png</normaloff>:/img/menu-edit-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/tribunesDef.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QQSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QQSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
